pkg/liqoctl/status/localinfo_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package status
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pterm/pterm"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	netv1alpha1 "github.com/liqotech/liqo/apis/net/v1alpha1"
+	"github.com/liqotech/liqo/pkg/liqoctl/factory"
+	"github.com/liqotech/liqo/pkg/liqoctl/output"
+	"github.com/liqotech/liqo/pkg/utils/testutil"
+)
+
+var _ = FDescribe("LocalInfo", func() {
+	const (
+		clusterID   = "fake"
+		clusterName = "fake"
+		namespace   = "liqo"
+	)
+
+	var (
+		clientBuilder fake.ClientBuilder
+		lic           *LocalInfoChecker
+		licTest       *LocalInfoChecker
+		ctx           context.Context
+		options       Options
+	)
+
+	BeforeSuite(func() {
+		ctx = context.Background()
+		_ = netv1alpha1.AddToScheme(scheme.Scheme)
+		clientBuilder = *fake.NewClientBuilder().WithScheme(scheme.Scheme)
+		clientBuilder.WithObjects(
+			testutil.FakeClusterIDConfigMap(namespace, clusterID, clusterName),
+			testutil.FakeIPAM(namespace),
+		)
+		options = Options{Factory: factory.NewForLocal()}
+		options.Printer = output.NewFakePrinter(GinkgoWriter)
+		options.CRClient = clientBuilder.Build()
+		pterm.DisableStyling()
+	})
+
+	FContext("Creating a new LocalInfoChecker", func() {
+		BeforeEach(func() {
+			licTest = &LocalInfoChecker{
+				localInfoSection: output.NewRootSection(),
+				options:          &options,
+				getReleaseValues: func() (map[string]interface{}, error) {
+					return testutil.FakeHelmValues(), nil
+				},
+			}
+		})
+		JustBeforeEach(func() {
+			lic = newLocalInfoChecker(&options)
+		})
+		It("should return a valid LocalInfoChecker", func() {
+			Expect(lic.localInfoSection).To(Equal(licTest.localInfoSection))
+			Expect(*lic.options).To(Equal(*licTest.options))
```

Comment:
```suggestion
			Expect(lic.localInfoSection).To(Equal(output.NewRootSection()))
			Expect(lic.options).To(PointTo(Equal(options)))
```

Then, you can drop the licTest structure.